docs/tutorial/getting-started.rst

```diff
@@ -14,20 +14,23 @@ Prerequisites
 
 * Ubuntu 22.04.
 * An Internet connection.
-* DSS relies on `MicroK8s`_, that requires as little as 540MB of memory.
-  But to accommodate workloads, we recommend a system with at least 20G
-  of disk space and 4G of memory.
+* `Snap`_.
```

Comment:
Should we have a `.` at the end of each requirement here?

Reply:
We don't have an agreement yet around this in the style guide. My usual (and pretty standard) approach is we use period if it is a sentence. No period if it's just one word/link. In the case of a list of bullet points, if there is at least one sentence per list (it uses period), then all bullet points goes with period at the end.

Reply:
ACK! As long as we follow it everywhere